libs/libRocket/Include/Rocket/Core/Element.h

```diff
@@ -524,7 +524,7 @@ class ROCKETCORE_API Element : public ScriptInterface
 	bool HasChildNodes() const;
 
 	/// Get a child element by its ID.
-	/// @param[in] id Id of the the child element
+	/// @param[in] id Id of the child element
```

Comment:
THe changes to librocket need to be made in the librocket repo or they will be lost next time I sync the files over.